examples/with-grafana/prometheus/prometheus.yml

```diff
@@ -0,0 +1,7 @@
+global:
```

Comment:
In addition to the Grafana dashboard, we should add an alert example, and We should show Grafana dashboard on our README